maintainers/maintainer-list.nix

```
@@ -1736,6 +1736,12 @@
     githubId = 11946442;
     name = "Dipin Hora";
   };
+  dirkx= {
```

```suggestion
  dirkx = {
```
nitpick: missed a space.

---

fixed.